src/Altinn.Profile/Program.cs

```diff
@@ -268,10 +270,8 @@ void Configure()
         // Enable higher level of detail in exceptions related to JWT validation
         IdentityModelEventSource.ShowPII = true;
     }
-    else
-    {
-        app.UseExceptionHandler("/profile/api/v1/error");
-    }
+    
+    app.UseExceptionHandler("/profile/api/v1/error");
```

Comment:
We could potentially keep the else and instead add
app.UseDeveloperExceptionPage(); to the if block above.